e2e/.env

```diff
@@ -0,0 +1,85 @@
+DEV_MODE=true
+LOG_LEVEL=debug
+TRUST_PROXY=true
+AUTHGEAR_APP_ID=accounts
+AUTHGEAR_CLIENT_ID=portal
+AUTHGEAR_ENDPOINT=http://accounts.portal.localhost:3100
+
+DATABASE_SCHEMA=public
+DATABASE_URL=postgres://postgres:postgres@localhost:5432/postgres?sslmode=disable
+
+AUDIT_DATABASE_SCHEMA=public
+AUDIT_DATABASE_URL=postgres://postgres:postgres@localhost:5432/postgres?sslmode=disable
+
+REDIS_URL=redis://localhost/0
+
+ANALYTIC_REDIS_URL=redis://localhost/1
+ANALYTIC_ENABLED=true
+ANALYTIC_EPOCH=2021-03-25
+
+ELASTICSEARCH_URL=http://localhost:9200
+
+CORS_ALLOWED_ORIGINS=portal.localhost:8000
+
+APP_HOST_SUFFIX=.localhost:3100
+APP_HOST_SUFFIXES=
+
+NFT_INDEXER_API_ENDPOINT=http://127.0.0.1:8080
+DENO_ENDPOINT=http://127.0.0.1:8090
+
+# Enable local fs config source
+CONFIG_SOURCE_TYPE=local_fs
+CONFIG_SOURCE_DIRECTORY=./var
+
+# Enable database config source
+#CONFIG_SOURCE_TYPE=database
+#APP_CUSTOM_RESOURCE_DIRECTORY./var
+#CUSTOM_RESOURCE_DIRECTORY=./var
+
+# This is for portal to create k8s ingress when creating domains
+#DOMAIN_IMPLEMENTATION=kubernetes
+#KUBERNETES_KUBECONFIG=./hack/kube-apiserver/.kubeconfig
+#KUBERNETES_APP_NAMESPACE=authgear
+#APP_KUBERNETES_INGRESS_TEMPLATE_FILE=./hack/ingress.tpl.yaml
+
+#SMTP_HOST=
+#SMTP_PORT=
+#SMTP_USERNAME=
+#SMTP_PASSWORD=
+
+#MAIL_SENDER=
+#MAIL_REPLY_TO=
+
+#STRIPE_SECRET_KEY=
+
+# Google Tag Manager container id for portal
+#GTM_CONTAINER_ID=
+
+# Portal frontend Sentry
+#PORTAL_FRONTEND_SENTRY_DSN=
+
+SEARCH_ENABLED=true
+WEB3_ENABLED=true
+AUDIT_LOG_ENABLED=true
+APP_DEFAULT_PLAN=free
+
+################################################################################
+
+COMPOSE_PROJECT_NAME=authgear-server-e2e
+
+MAIN_LISTEN_ADDR=0.0.0.0:4000
+RESOLVER_LISTEN_ADDR=0.0.0.0:4001
+ADMIN_LISTEN_ADDR=0.0.0.0:4002
+
+MAIN_INTERNAL_LISTEN_ADDR=0.0.0.0:14000
+RESOLVER_INTERNAL_LISTEN_ADDR=0.0.0.0:14001
+ADMIN_INTERNAL_LISTEN_ADDR=0.0.0.0:14002
+
+DATABASE_URL=postgres://postgres:postgres@db:5432/postgres?sslmode=disable
+AUDIT_DATABASE_URL=postgres://postgres:postgres@db:5432/postgres?sslmode=disable
+
+REDIS_URL=redis://redis:6379/0
```

Comment:
It seems that this section of the file overrides the variables declared above. To avoid confusion, remove or comment out the overridden variables above.

Reply:
Should remove from run.sh and Makfile